.travis.yml

```diff
@@ -32,7 +32,8 @@ install:
 - ls $SSL_CERT_DIR
 
 script:
-- nosetests -vv --traverse-namespace proteus/tests proteus/tests/ci  proteus/tests/poisson_2d
+- nosetests -vv --traverse-namespace proteus/tests proteus/tests/ci  proteus/tests/poisson_2d  proteus/tests/mesh_tests -a '!slow'
```

Comment:
Okay, I see you just add the @attribute('fast',...) or @attribute('slow',...) to each test to classify into fast and slow tests. Is that the idea? The makes organizing the tests into tests for travis  and other CI frameworks (e.g. buildbot) much less invasive than using a directory hierarchy.